src/Arcus.Messaging.Tests.Unit/MessageHandling/ServiceBus/AzureServiceBusMessageRouterTests.cs

```diff
@@ -146,9 +146,9 @@ public async Task RouteMessage_WithMessageHandlerMessageBodyFilter_GoesThroughRe
             var ignoredHandler = new StubServiceBusMessageHandler<Order>();
             var spyHandler = new StubServiceBusMessageHandler<Order>();
             collection.WithServiceBusMessageHandler<StubServiceBusMessageHandler<Order>, Order>(
-                          messageBodyFilter: body => true, implementationFactory: serviceProvider => spyHandler)
+                          implementationFactory: serviceProvider => spyHandler, opt => opt.AddMessageBodyFilter(body => true))
                       .WithServiceBusMessageHandler<StubServiceBusMessageHandler<Order>, Order>(
-                          messageBodyFilter: body => false, implementationFactory: serviceProvider => ignoredHandler);
+                          implementationFactory: serviceProvider => ignoredHandler, opt => opt.AddMessageBodyFilter(body => false));
```

Comment:
We're not doing anything with `body`, so I believe we can use the discard operator here ?  
```
_ => true
```
and
```
_ => false
``` 
?

Reply:
Done so here: https://github.com/arcus-azure/arcus.messaging/pull/590/commits/0aef8e8f0dcdbe3977fc185e43c3150792a630d0